### ISSUE TITLE
SSL doesn't work on Raspberry Pi

### ISSUE THREAD

**Issue by lkalir** [lkalir/trashbot#9]
The new `wha_happun` command doesn't actually run on the Pi. When the graphql query actually comes out we see
```
[2021-03-04T07:40:47Z ERROR trashbot::commands] Command 'wha_happun' returned error reqwest::Error { kind: Request, url: Url { scheme: "https", username: "", password: None, host: Some(Domain("api.github.com")), port: None, path: "/graphql", query: None, fragment: None }, source: hyper::Error(Connect, Ssl(Error { code: ErrorCode(1), cause: Some(Ssl(ErrorStack([Error { code: 337047686, library: "SSL routines", function: "tls_process_server_certificate", reason: "certificate verify failed", file: "../ssl/statem/statem_clnt.c", line: 1913 }]))) }, X509VerifyResult { code: 20, error: "unable to get local issuer certificate" })) }
```

**Comment by lkalir** [comment]
looks like we just need to install `ca-certificates` and run `update-ca-certificates` at image creation and it should be good

**Comment by lkalir** [comment]
Fixed as of 8e712fb0a35d31e7202bcc84b91d836f287fcea8
